src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -183,4 +193,149 @@ public static String getUserEmail(String userId)
 		}
 		return remoteUserEmail;
 	}
+	
+	/************* LDAP lookup ****************/
+	private static final String OU_GROUPS = "ou=groups";
+	private static final String USE_TESTUSER_URL = "use.testUser.url";
```

Comment:
How about calling the property itself something like 'use.testUser.urlOverride' or 'allow.testUser.urlOverride'?  From the current name I'd expect the value to actually be an URL.